README.md

```diff
@@ -178,7 +178,8 @@ class CustomStyle: DayStyle {
 
     override func apply() {
         super.apply()
-        BottomBannerView.appearance().backgroundColor = .orange
```

Comment:
It'd be better to move styling changes to separate PR, but since they're so small and more or less related (mostly docs), I merged them into one PR.